.github/workflows/cmake.yml

```diff
@@ -0,0 +1,69 @@
+name: CMake
+
+on:
+  push:
+    branches: 
+      - main
+  pull_request:
+    branches:
+      - main
+
+env:
+  # Customize the CMake build type here (Release, Debug, RelWithDebInfo, etc.)
+  BUILD_TYPE: Release
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+          - windows-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install boost
```

Comment:
This step causes warnings in both Ubuntu and Windows. Can we fix or suppress (if appropriate)?
![image](https://user-images.githubusercontent.com/3143819/204975702-1771c77e-6400-4e3d-9cb3-1bc80ea3a87d.png)